hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/compact/DeserializedGenericRecordBuilder.java

```diff
@@ -32,27 +33,34 @@
  */
 public class DeserializedGenericRecordBuilder extends AbstractGenericRecordBuilder {
 
-    private final TreeMap<String, Object> objects = new TreeMap<>();
+    private final TreeMap<String, Object> objects;
```

Comment:
just wondering, any reason for this change?

Reply:
Just to initialize everything in the constructor.